### PR TITLE
Added .arc settings and submit for Zeus

### PR DESCRIPTION
**Pull request by alongd**
Currently only supporting Gaussian

### CODE REVIEW

File: Servers/Zeus/ARC/.arc/settings.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+"""
+
+import os
+import string
+
+
+servers = {
+    'local': {  # Each Zeus node containes 80 cores and 378 GB RAM
+        'cluster_soft': 'PBS',
+        'un': 'alon',
```
Comment (kfir4444) [inline Servers/Zeus/ARC/.arc/settings.py:13]: Maybe in the places users need to fill up their own info, you add something like a variable.

File: Servers/Zeus/ARC/.arc/submit.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+"""
+Submit scripts
+sorted in a dictionary with server names as keys
+"""
+
+
+submit_scripts = {
```
Comment (kfir4444) [inline Servers/Zeus/ARC/.arc/submit.py:7]: I would be great if you could also add the molpro script, as it is coming soon

File: Servers/Zeus/ARC/.arc/submit.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+GAUSS_SCRDIR=/home/{un}/scratch/g09/$PBS_JOBID
+export $GAUSS_SCRDIR
+mkdir -p $GAUSS_SCRDIR
+
+. ~/.bashrc
+
+which g09
+
+touch initial_time
```
Comment (kfir4444) [inline Servers/Zeus/ARC/.arc/submit.py:28]: These are normally empty files..

**Review by kfir4444 — CHANGES_REQUESTED** [review]
Thanks @alongd,
Please see the following changes

File: Servers/Zeus/ARC/.arc/submit.py (hunk 2 above)
Comment (alongd) [reply Servers/Zeus/ARC/.arc/submit.py:7]: I first want to test it to see that it works... then will upload a working version

File: Servers/Zeus/ARC/.arc/submit.py (hunk 3 above)
Comment (alongd) [reply Servers/Zeus/ARC/.arc/submit.py:28]: correct. It helps ARC keep track of the execution time, it's not reported accurately by the server or the ESS

File: Servers/Zeus/ARC/.arc/settings.py (hunk 1 above)
Comment (alongd) [reply Servers/Zeus/ARC/.arc/settings.py:13]: The server knows your the username (`$USER`), sometimes in ARC we use that. I'm not sure it is used throughout. Once we fix that in ARC we can remove the 'un' here altogether

Comment (kfir4444) [reply Servers/Zeus/ARC/.arc/settings.py:13]: Ok, so could you just comment so that people will now to replace it?

**Review by kfir4444 — APPROVED** [review]

**Pull request merged**